org.bluej.BlueJ.json

```diff
@@ -0,0 +1,73 @@
+{
+  "app-id" : "org.bluej.BlueJ",
+  "branch" : "4.2.0",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "3.30",
+  "sdk" : "org.gnome.Sdk",
+  "sdk-extensions" : [ "org.freedesktop.Sdk.Extension.openjdk11" ],
+  "command": "org.bluej.BlueJ",
+  "modules" : [
+    {
+      "name" : "openjdk",
+      "buildsystem" : "simple",
+      "build-commands" : [ "/usr/lib/sdk/openjdk11/install.sh" ]
+    },
+    {
+      "name" : "javafx",
+      "buildsystem" : "simple",
+      "build-commands": ["mkdir -p /app/javafx", "cp -r ./ /app/javafx"],
+      "sources": [{
+        "type": "archive",
+        "url": "https://download2.gluonhq.com/openjfx/11.0.2/openjfx-11.0.2_linux-x64_bin-sdk.zip",
+        "sha256": "40ef06cd50ea535d45403d9c44e9cb405b631c547734b5b50a6cb7b222293f97"
+      }]
+    },
+    {
+      "name": "bluej",
+      "buildsystem": "simple",
+      "build-commands": [
+        "unzip bluej-dist.jar -d dist",
+        "mkdir -p /app/bluej",
+        "cp -r dist/* /app/bluej",
+        "for size in 32 48 256; do install -Dm644 /app/bluej/lib/images/bluej-icon-${size}.png /app/share/icons/hicolor/${size}x${size}/apps/org.bluej.BlueJ.png; done"
+      ],
+      "sources": [{
+        "type": "archive",
+        "url": "http://www.bluej.org/download/files/BlueJ-generic-420.jar",
+        "dest-filename": "BlueJ-generic-420.zip",
+        "sha256": "3d10306340f74e22885f2a5f939cb1b19cbc2e048c9e6e22418627ae5833c56f"
+      }]
+    },
+    {
+      "name": "bluej-startup",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dm755 bluej /app/bin/org.bluej.BlueJ",
+        "install -Dm644 org.bluej.BlueJ.desktop /app/share/applications/org.bluej.BlueJ.desktop",
+        "install -Dm644 org.bluej.BlueJ.appdata.xml /app/share/metainfo/org.bluej.BlueJ.appdata.xml"
+      ],
+      "sources": [{
+        "type": "file",
+        "path": "bluej"
+      },{
+        "type": "file",
+        "path": "org.bluej.BlueJ.desktop"
+      },{
+        "type": "file",
+        "path": "org.bluej.BlueJ.appdata.xml"
+      }]
+    }
+  ],
+  "finish-args" : [
```

Comment:
could you put this at the top of the file, just before the modules? no particular reason other than that basically everything else that isn't autogenerated does it that way and it's a bit easier to read and review.